src/api/route-services/projects.js

```diff
@@ -0,0 +1,42 @@
+const config = require('../../config');
+const {
+  createDynamoDbInstance, convertToDynamoDbRecord,
+} = require('../../utils/dynamoDb');
+
+const { OK, NotFoundError } = require('../../utils/responses');
+
+class ProjectsService {
+  constructor() {
+    this.tableName = `projects-${config.clusterEnv}`;
+  }
+
+  async updateProject(projectUuid, project) {
```

Comment:
do we have project data copied across different tables? Because if we do, then we will need to update all places where we store projects, so that tables don't get out of sync

Reply:
No, `project` data is only in `projects` table